loops_lab.js

```diff
@@ -0,0 +1,208 @@
+// # Loops Lab Exercises
+//-----------------------
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+//  A)
+// let num = 12;
+// while (num >= 1) {
+//   console.log(num);
+//     num -= 1;
+// }
+ // B)
+// for (let num = 12; num >= 1; num --) {
+//   console.log(num);
+// }
+// //
+// ---------------------------------------------------------------------------------------------------------------------------------------------------
+// 2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "max".
+// For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+//
+//  A)
+// let max = 0;
+// while (max <=15) {
+//     if (max % 2 === 0){
+//        console.log(max + ' is even');
+//      } else {
+//        console.log(max + ' is odd');
+//      }
+//      max++
+//     }
+//
+//  B)
+// for (let max = 0; max <= 15; max += 1 ){
+//       if ( max % 2 == 0) {
+//         console.log(max + ' is even');
+//           } else {
+//         console.log( max + ' is odd');
+//       }
+//     }
+// ---------------------------------------------------------------------------------------------------------------------------------------------------
+// 3. Write a `while` loop and a `for` loop that takes the variable "x" and iterates over all numbers from 0 to "x".
+// For each iteration of the loop, it will multiply the number of "x" by 9 and log the result (e.g. "2 * 9 = 18").
+//   _Bonus_ think of another way to solve it.
+//  A)
+// let cst = 9;
+// let x = 0;
+// while (x <= 10){
+//   console.log( cst  + " * " + x + " = " + x * cst );
+//   x += 1;
+// }
+//
+ // B)
+//  let cst = 9;
+//  let num = 0;
+// for(let num = 0; num <= 10; num ++ ){
+//   console.log(cst  + " * " + num + " = " + num * cst)
+// }
+// ---------------------------------------------------------------------------------------------------------------------------------------------------
+// 4. Write code that prints/logs all the numbers that end in a 5 from 1 to 100, **exclusive**.
+// for (let x = 5; x <= 100 ; x += 10){
```

Comment:
What would happen if x starts from 0 instead of 5? Would there be better way to solve this by using modulo?